examples/redis-mysql/associations.test.js

```diff
@@ -0,0 +1,86 @@
+const Redis = require('ioredis')
```

Comment:
Examples folder is purely for examples. Tests placed there will not be executed by ci. Please align with existing tests.